packages/core/src/messages/packet/index.ts

```diff
@@ -222,11 +223,10 @@ export class Packet<Chain extends HoprCoreConnector> extends Uint8Array {
         node._interactions.payments.onChainKey.interact(path[0])
       )
 
-      const newFee = new node.paymentChannels.types.Balance(100)
+      const ticketAmount = new BN(String(node.ticketAmount)).add(fee)
```

Comment:
@robertkiel here we are adding the relay fees, does it look right to you?

Reply:
This is not completely true. It should say:
```
ticketAmount = ( (pathLength - 1) * relayFee ) / winProb
```